src/test/ui/parser/raw/raw-str-unterminated.stderr

```diff
@@ -2,9 +2,7 @@ error: unterminated raw string
   --> $DIR/raw-str-unterminated.rs:2:5
    |
 LL |     r#" string literal goes on
-   |     ^ unterminated raw string
-   |
-   = note: this raw string should be terminated with `"#`
```

Comment:
Maybe we should keep the current `note` for this case where no likely close is found.